js/interface.js

```diff
@@ -359,7 +360,11 @@ function fetchCurrentDataSourceEntries(entries) {
     } else {
       var flattenedColumns = {};
 
-      rows.map(({ data }) => data).forEach(dataItem => (flattenedColumns = { ...flattenedColumns, ...dataItem }));
+      rows.map(function(row) {
```

Comment:
@squallstar Can you confirm if this is still correctly optimised based on the work done in https://github.com/Fliplet/fliplet-widget-data-source/pull/177

Reply:
@tonytlwu is there a specific reason you want to change the current functionality not to use the spread operator?

Your solution seems faster but creates almost twice as many operations?

Reply:
@squallstar Because it isn't allowed for ES5 and our ESLint has not been updated to support this.

Reply:
I think high ops/s is meant to be a good thing though.